crates/ty_python_semantic/src/types/display.rs

```diff
@@ -605,7 +604,6 @@ impl Display for DisplayLiteralGroup<'_> {
 /// For example, `Literal[1] | Literal[2] | Literal["s"]` is displayed as `"Literal[1, 2, "s"]"`.
 #[derive(Copy, Clone, Debug, Eq, PartialEq, Hash)]
 enum CondensedDisplayTypeKind {
```

Comment:
I don't think we are going to add more of these, so I think we could remove this enum entirely and simplify the literal display code to not need to consider multiple "kinds" at all.

But I'm not going to hold up this PR for that, gonna land this quick before there are any conflicts.